pageserve/test_routes.py

```diff
@@ -240,8 +240,7 @@ def test_add_valid_post(self, mock_requests):
 
         response = self.client.post('/v1/add_post', data=VALID_POST_FORM)
 
-        self.assertEqual(response.status_code, 201)
-        self.assertEqual(response.data.decode(), "Example success message")
+        self.assertEqual(response.status_code, 302)  # redirect to index
```

Comment:
Check the location header?

Reply:
Done https://github.com/knative-portability/large-events/pull/61/commits/489d18ff8668d2620fa62e22460f856565a08dad